pingcap/tidb
# metrics: make client traffic simply

Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>



### What problem does this PR solve?


Issue Number: close #31115

Problem Summary:
When there are many tidb servers, client traffic metrics is too messy.

![image](https://user-images.githubusercontent.com/4352397/147068730-fa6c40ab-9d7d-452c-898a-64df5c036b5a.png)

### What is changed and how it works?

Sum client traffic by tidb servers

### Check List

Tests 

1. start a cluster
2. check client traffic metrics

<img width="675" alt="截屏2021-12-22 下午4 05 32" src="https://user-images.githubusercontent.com/4352397/147068654-a135ba16-1e20-4038-aac3-2866329e4cb0.png">


Side effects


Documentation


### Release note



```release-note
None
```
